ConsoleTest/Program.cs

```diff
@@ -1,104 +1,64 @@
-        //    {
-        //        Thread.Sleep(1);
-        //        system.update();
+                    Console.WriteLine("--== Test Menu ==--");
+                    Console.WriteLine("1. Play Sound");
+                    Console.WriteLine("2. Play Bank");
 
-        //        float time = (float)(DateTime.Now.TimeOfDay.TotalSeconds * 0.025f) % 1;
-        //        Console.Title = time.ToString();
-        //        timeParam.setValue(time).Check();
-        //    }
-        //}
+                    Console.WriteLine("Q. Quit");
 
-        public Program()
-        {
-            using (SupersonicSound.Studio.System system = new SupersonicSound.Studio.System(preInit: PreInit))
-            {
-                var master = system.LoadBankFromFile("Master Bank.bank", BankLoadingFlags.Normal);
-                var strings = system.LoadBankFromFile("Master Bank.strings.bank", BankLoadingFlags.Normal);
-                var bank = system.LoadBankFromFile("Surround_Ambience.bank", BankLoadingFlags.Normal);
+                    string choice = Console.ReadLine().Trim().ToLower();
+                    if (choice == "q")
+                        break;
 
-                var loopingAmbienceDescription = system.GetEvent("event:/Ambience/Country");
-                var loopingAmbienceInstance = loopingAmbienceDescription.CreateInstance();
-                var timeParam = loopingAmbienceInstance.Parameters["Time"];
+                    switch (choice)
+                    {
+                        case "1":
+                            new PlaySound(system, contentPath).Execute();
+                            break;
 
-                loopingAmbienceInstance.Start();
+                        case "2":
+                            new PlayBank(system, contentPath).Execute();
+                            break;
 
-                while (!Console.KeyAvailable)
-                {
-                    Thread.Sleep(1);
-                    system.Update();
+                        default:
+                            Console.WriteLine("Unknown choice");
+                            break;
+                    }
 
-                    float time = (float)(DateTime.Now.TimeOfDay.TotalSeconds * 0.025f) % 1;
-                    Console.Title = time.ToString();
-                    timeParam.Value = time;
+                    Console.WriteLine();
                 }
 
-                loopingAmbienceInstance.Stop(true);
-
-                while (true)
-                {
-                    Thread.Sleep(1);
-                    system.Update();
-                    Console.Title = loopingAmbienceInstance.PlaybackState.ToString();
-                }
+                Console.WriteLine("Shutting down");
             }
+
+            Console.WriteLine("Closed");
         }
 
         private static void PreInit(IPreInitilizeLowLevelSystem ll)
         {
-            ll.Output = OutputMode.DirectSound;
+            if (!SupersonicSound.Wrapper.Util.IsLinux)
```

Comment:
This is a hack to address [issue 6](https://github.com/martindevans/SupersonicSound/issues/6).

If you have a windows dev box, can you duplicate this? (see issue for more details)